script.js

```diff
@@ -16,6 +16,9 @@ const sheetId = "1AkYjbnLbWW83LTm6jcsRjg78hRVxWsSKQv1eSssDHSM";
 // the URI that grabs the sheet text formatted as a CSV
 const sheetURI = `https://docs.google.com/spreadsheets/d/${sheetId}/export?format=csv&id=${sheetId}`;
 
+// the URI for CARTO counties layer joined to the moratoriums data (both in CARTO  acct)
+const cartoCountiesURI = createCartoURI();
+
 // states geojson url
 const statesGeoJsonURI = "./states.geojson";
```

Comment:
@timhitchins let's try the same approach with the states layer as the counties layer, using this table in Carto:
https://ampitup.carto.com/dataset/states

It would alleviate the need for the states data join client side.